sandbox/simulator/randomtext

```diff
@@ -90,6 +90,15 @@ function GenerateRandomPostText($maxParagraphs = 8)
 		$output .= YoutubeRandomVideo();
 	}
 
+	// Add link to the post
+	global $uselinks;
+	global $linkchance;
+	if ($uselinks and random_int(0, $linkchance-1) == 0)
```

Comment:
Just to be sure, is there any reason `and` is used here instead of `&&`?

Reply:
I didn't realize "and" was even an option, huh. I'd prefer && though.

Reply:
There was no reason for it to be `and` over `&&` so i changed it.